site/hack/release-docs.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# How to run the command to generate new documentation of impkg from develop with the version v0.5.0
+# ./hack/release-docs.sh imgpkg v0.5.0
+
+TOOL=$1
+NEW_VERSION=$2
+FROM_VERSION=${3:-develop}
+
+CONTENT_DIRECTORY=content/${TOOL}/docs
+TOC_DIRECTORY=data/${TOOL}/docs
+
+# don't run if there's already a directory for the target docs version
+if [[ -d $CONTENT_DIRECTORY/$NEW_VERSION ]]; then
+    echo "ERROR: $CONTENT_DIRECTORY/$NEW_VERSION already exists"
+    exit 1
+fi
+
+LATEST_VERSION=$(yq e ".params.${TOOL}.version_latest" config.yaml)
```

Comment:
Running this locally,

```console
$ ./hack/release-docs.sh ytt v0.39.0
usage: yq [-h] [--yaml-output] [--yaml-roundtrip] [--width WIDTH]
          [--indentless-lists] [--in-place] [--version]
          [jq_filter] [files ...]
yq: error: argument files: can't open '.params.ytt.version_latest': [Errno 2] No such file or directory: '.params.ytt.version_latest'
```

I suspect I've got the _other_ `yq` installed (the one that wraps `jq`)?

FWIW, as inelegant as it is, this works (and we're already dependent on `ytt`, so one less dependency?):

```suggestion
LATEST_VERSION=$(ytt -f version.yml=<(echo -e "#@ load('@ytt:data', 'data')\n--- #@ data.values.params.${TOOL}.version_latest") --data-values-file config.yaml)
```

Reply:
done